docs/_pages/releases.md

```diff
@@ -9,6 +9,11 @@ sidebar:
 
 ## Unreleased
 
+### What's new
+* Add `BeDefined` and `NotBeDefined` to `EnumAssertions` to assert on existence of an enum value - [#1888](https://github.com/fluentassertions/fluentassertions/pull/1888)
```

Comment:
🔧 I would remove the `EnumAssertions` reference. Consumers of the library don't know that it exists.